### PR TITLE
Governance and open source infrastructure

### PR DESCRIPTION
**Pull request by bkeepers**
Here's my initial thoughts on one of the issues in the NPM debacle this week.

There's a lot more I want to say here, especially about foundations, distribution, and federation vs centralization, but it would be weeks before I get it out. _Update:_ I decided to take some time to say more. 
#### [Vew Rendered Post](https://github.com/bkeepers/bkeepers.github.io/blob/governance/_posts/2016/2016-03-25-governance.md)

I'd love any thoughts, challenges, or feedback.

/cc @mlinksva @arfon @benbalter

### CODE REVIEW

File: _posts/2016/2016-03-25-governance.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+But "governance" can apply to things besides nation states. My childhood home had a two-party system that created and enforced the policies that governed our land, money, and citizens. This system was ruled by my mom who weilded most of the power, including the ability to unilaterally create new policies, enforce law and order, rule on disputes amongst the citizens, and imprison anyone on a whim. My dad technically had veto power over everything, but he rarely exercised it due to the fear of political repercussions. None of this was written in law books, but there was an inherent governance system.
+
+Governance is especially important when shared resources are subject to rivalry–what economists call _rivalrous_. If I can interfere with your use of a resource, then it is rivalrous. There are many ways governance deals with rivalrous resources, such as charing you to use them, penalizing you for abusing them, or assigning governance of specific things to others.
```
Comment (mlinksva) [inline _posts/2016/2016-03-25-governance.md]: tautological first sentence. maybe 'resources cannot be enjoyed simultaneously by multiple parties', or 'resources are depleted with each use', or ' resources have a marginal production cost greater than zero'...

Second sentence is not quite right. If my use of a resources can interfere with your use... My ability to interfere just says I have power, not necessarily anything about the resource.

Comment (arfon) [inline _posts/2016/2016-03-25-governance.md]: Typo: ~~charing~~ -> charging

File: _posts/2016/2016-03-25-governance.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Property is a form of governance that assigns ownership as a way of dealing with rivalrous resources. You are given exlusive ownership of a resource, and it's up to you to decide who else can use it and what they can do with it.
```
Comment (arfon) [inline _posts/2016/2016-03-25-governance.md]: ~~exlusive~~ exclusive

**Comment by arfon** [comment]
I like this, a lot. Corrected a couple of typos but I say :shipit:

File: _posts/2016/2016-03-25-governance.md (hunk 1 above)
Comment (benbalter) [inline _posts/2016/2016-03-25-governance.md]: > such as charing you to use them

Charging?

**Comment by benbalter** [comment]
It's good and got me thinking. Don't think this should change anything, but counterpoint for sake of discussion, that projects for which the community feels a sense of ownership (or at least entitlement) is an edge case. The *vast* majority of the corpus of open source (see e.g., https://github.com/benbalter for one example), is *my* in both senses, not because I'm unwilling to relinquish ownership or exclusive control, but because nobody else has any interest in interfering with my exclusive rights. I'd caution against any proposal that weakens the bond between owner and project for the vast majority of projects (those that are individual-, not community-owned), as that might increase the amount of abandonware in the world (how often have you found a great project, only to see it's GitHub repo hasn't been updated in four years?). Put another way, this is a problem for maybe 1% of open source projects? Maybe less? Do you optimize the system for the most used, or for the largest by volume (which may further raise the barrier to get to that 1% for smaller projects)?

**Pull request closed without merging**